common/src/main/resources/less/style.less

```diff
@@ -951,6 +951,18 @@ tfoot {
   }
 }
 
+.constraints-grid {
+  padding: 0.5em;
+
+  .elevation-range-picker {
+    width: 18ex;
```

Comment:
Any reason to prefer ex over em?

Reply:
Hmmm. No. I think this was a neural meltdown. I'll change to em.